runtime/gc_vlhgc/AllocationContextBalanced.cpp

```diff
@@ -735,7 +735,7 @@ MM_AllocationContextBalanced::acquireMPBPRegionFromContext(MM_EnvironmentBase *e
 			if (region->_allocateData.taskAsMemoryPoolBumpPointer(env, requestingContext)) {
```

Comment:
taskAsMemoryPoolBumpPointer should be renamed

Reply:
rename to taskAsMemoryPoolAddressOrderedList